packages/flutter/lib/src/material/tooltip_theme.dart

```diff
@@ -250,3 +271,18 @@ class TooltipTheme extends InheritedTheme {
   @override
   bool updateShouldNotify(TooltipTheme oldWidget) => data != oldWidget.data;
 }
+
+/// The mode that will decide when the tooltip will trigger
```

Comment:
```suggestion
/// The method of interaction that will trigger a tooltip.
```

Reply:
This should also cross link some of the related classes for those reading this in the api documentation, like `[Tooltip]`.